spec/features/sign_out_of_Chitter_spec.rb

```diff
@@ -0,0 +1,18 @@
+feature 'User signs out' do
```

Comment:
Style/FileName: Use snake_case for source file names. (https://github.com/bbatsov/ruby-style-guide#snake-case-files)